### ISSUE TITLE
abr antena

### ISSUE THREAD

**Issue by eplayerclick** [redmusicxd/iptv_ro#15]
For antenaplay, can the abr link be served, not specific qualtiy one

**Comment by eplayerclick** [comment]
how?

**Comment by redmusicxd** [comment]
Hi, please check in config.json if quality is set to auto, it was hq in the past and and I corrected in recently along with other fixes

**Comment by eplayerclick** [comment]
OK, if i do so it no longer curls ivm antenaplay link?

**Comment by redmusicxd** [comment]
That’s right, if it doesn’t work please make sure you are up to date with
the repository

**Comment by eplayerclick** [comment]
Ok also do you think you can add orange tv go?

**Comment by redmusicxd** [comment]
unfortunately orange could be added but streams are encrypted Widevine, for digi i made an exception because it has just a few of them like this and i can easily get the stream and decryption proxy url, as for primaplay yes i'll add it in the future as streams are not encrypted

**Comment by the-poor-gamer-124** [comment]
they use an iframe so for the beginning can that iframe be grabbed?

mie., 29 sept. 2021, 23:50 Dan Negru ***@***.***> a scris:

**Comment by redmusicxd** [comment]
guess that would be possible but for the moment am working on a "v2" version of the script among other things

**Comment by eplayerclick** [comment]
Ok
